tests/profiles/stat_test.py

```diff
@@ -0,0 +1,35 @@
+from fontbakery.checkrunner import (
+              DEBUG
+            , INFO
+            , WARN
+            , ERROR
+            , SKIP
+            , PASS
+            , FAIL
+            )
+from fontbakery.utils import (assert_PASS,
+                              assert_results_contain)
+
+check_statuses = (ERROR, FAIL, SKIP, PASS, WARN, INFO, DEBUG)
+
+from fontTools.ttLib import TTFont
+
+def test_check_varfont_stat_axis_record_for_each_axis():
+    """ Check the STAT table has an Axis Record for every axis in the font. """
+    from fontbakery.profiles.stat import com_google_fonts_check_varfont_stat_axis_record_for_each_axis as check
+
+    # Our reference varfont, Cabin[wdth,wght].ttf, has
+    # all necessary Axis Records
+    ttFont = TTFont("data/test/cabinvf/Cabin[wdth,wght].ttf")
+
+    # So it must PASS the test
```

Comment:
strictly speaking this should cause the check (not the test) to PASS ;-)

We **check** fonts. And we **test** the check's source code